src/modules/reading-time/index.es

```diff
@@ -0,0 +1,45 @@
+import cornea from "cornea"
+
+export default cornea.extend({
```

Comment:
I just quickly adapted old code. Anyway, like said in the PR below, I am going to make this "server" side.